packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/models/permissions_model.dart

```diff
@@ -0,0 +1,19 @@
+{{> licence.dart }}
+
+import 'package:json_annotation/json_annotation.dart';
+
+part 'permissions_model.g.dart';
+
+@JsonSerializable()
+class PermissionsModel {
```

Comment:
Please use the name `PermissionListModel`

Reply:
Done